src/main/java/dev/codesoapbox/dummy4j/dummies/shared/math/NumberValidator.java

```diff
@@ -33,4 +34,19 @@ public static void inRange(float value, float min, float max) {
             throw new ValueOutOfRangeException(value, min, max);
         }
     }
+
+    /**
+     * Verifies if the given value fits in the {@code min-max} range
+     * and if the {@code max} is greater than the {@code min}
+     *
+     * @throws InvalidRangeException    if the given {@code min} is greater than the given {@code max}
+     * @throws ValueOutOfRangeException if the given value is not within the {@code min-max} range
+     * @since SNAPSHOT
+     */
+    public static void inProperRange(float value, float min, float max) {
```

Comment:
See if this could entirely replace `inRange`, as it's essentially the same but better.